types/crack-json/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "rules": {
+        "@definitelytyped/no-declare-current-package": "off",
+        "@definitelytyped/no-single-declare-module": "off"
+    }
+}
```

Comment:
I'd appreciate some guidance if these exceptions are ok. I can't get it to work without `declare module "crack-json"`, but maybe I'm just doing it wrong